### PR TITLE
[EDE-988] upgrade themes to juniper

### PR DESCRIPTION
**Pull request by emZubair**
### [[EDE-988] upgrade themes to juniper](https://edlyio.atlassian.net/browse/EDE-988)

Theme upgraded to juniper.

### CODE REVIEW

**Review by arbirali — APPROVED** [review]

**Review by mumerfarooq000 — APPROVED** [review]
LGTM

File: mitx/lms/templates/dashboard/_dashboard_course_listing.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <%page args="course_overview, enrollment, entitlement, entitlement_session, course_card_index, is_unfulfilled_entitlement, is_fulfilled_entitlement, entitlement_available_sessions, entitlement_expiration_date, entitlement_expired_at, show_courseware_link, cert_status, can_refund_entitlement, can_unenroll, credit_status, show_email_settings, course_mode_info, is_paid_course, is_course_blocked, verification_status, course_requirements, dashboard_index, share_settings, related_programs, display_course_modes_on_dashboard, show_consent_link, enterprise_customer_name, resume_button_url" expression_filter="h"/>
 
 <%!
-import urllib
+import six
```
Comment (danialmalik) [inline mitx/lms/templates/dashboard/_dashboard_course_listing.html:4]: Why do we need `six` if we are upgrading to python3?

**Review by imhassantariq — APPROVED** [review]

File: mitx/lms/templates/dashboard/_dashboard_course_listing.html (hunk 1 above)
Comment (danialmalik) [reply mitx/lms/templates/dashboard/_dashboard_course_listing.html:4]: Figured out that edX is using `six` so no need to change this

**Review by danialmalik — APPROVED** [review]

File: mitx/lms/templates/courseware/course_about.html [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 <%!
-import math
-from datetime import datetime, timedelta
+from math import ceil
 from django.utils.translation import ugettext as _
 from django.utils.translation import pgettext
 from django.urls import reverse
-from courseware.courses import get_course_about_section
+from lms.djangoapps.courseware.courses import get_course_about_section
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
```
Comment (mumarkhan999) [inline mitx/lms/templates/courseware/course_about.html]: Normally, in openedx we sort the imports in this fashion
1 system-level imports like `import io`
`#` blank line
2 installed packages imports like `from django.conftrib.auth.models import User`
`#` blank line
3 project-level imports like `from openedx.core.djangoapps.content.course_overviews.models import CourseOverview`

**Pull request merged**